### PR TITLE
Added configuration for Docker

### PR DESCRIPTION
**Pull request by hakamadare**
I've written a Dockerfile and Docker Compose configuration to enable quick
deployment of a SFTP service based on the sample server.

### CODE REVIEW

**Comment by cmrigney** [comment]
Thanks for the PR! I'll try it out.

File: README.md [hunk 1]
```diff
@@ -55,3 +55,13 @@ The path parameter includes the user's subfolder name.
...
 Password authentication is NOT currently supported.  Only public/private key auth. It should be simple to add this feature.  A PR is welcome. :)
+
+### Docker
+
+The module also includes a server suitable for running in a Docker container.
+
+1. `cp .env.sample .env`
+2. Edit `.env` to add appropriate configuration values for all variables
+3. `mkdir keys && ssh-keygen -t rsa -f keys/server_key_rsa`
+4. For each user, `ssh-keygen -t rsa -f keys/id-<username>`
```
Comment (cmrigney) [inline README.md]: I had to change `id-<username>` to `id_<username>`

**Comment by cmrigney** [comment]
Works great! Just had to make that one change mentioned.  Could you fix that and I will merge?

File: README.md (hunk 1 above)
Comment (hakamadare) [inline README.md]: d'oh, good catch :)

**Comment by hakamadare** [comment]
@cmrigney fixed!

**Comment by cmrigney** [comment]
Thanks!

**Pull request merged**